BossMod/QuestBattle/QuestBattleConfig.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace BossMod;
+
+[ConfigDisplay(Name = "Quest Battles (Solo Duties)", Order = 6)]
```

Comment:
Is it fair to say that this is basically a zone-wide 'modules'? They could e.g. be used to path in dungeons a-la autoduty.

Reply:
yeah i actually experimented with that for Antitower and Xelphatol, it was in some ways superior to autoduty since i had access to all the worldstate stuff easily